src/Microsoft.OpenApi/Any/OpenApiAnyCloneHelper.cs

```diff
@@ -32,5 +32,27 @@ public static IOpenApiAny CloneFromCopyConstructor(IOpenApiAny obj)
 
             return obj;
         }
+        
+        /// <summary>
+        /// Clones an instance of <see cref="IOpenApiAny"/> object from the copy constructor
+        /// </summary>
+        /// <param name="obj">The object instance.</param>
+        /// <returns>A clone copy or the object itself.</returns>
+        public static IOpenApiAny CloneFromCopyConstructor<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)] T>(T obj) where T : IOpenApiAny
```

Comment:
can you tag the non generic one as obsolete so we have a better idea about which one is binding in which scenario please?